src/plugins/intel_cpu/src/nodes/kernels/stack_allocator.hpp

```diff
@@ -393,11 +393,12 @@ class StackAllocator::RegAddress : public StackAllocator::Address {
 
 private:
     static size_t getAlignment() {
-        using namespace dnnl::impl::cpu::x64;
+        namespace x64 = dnnl::impl::cpu::x64;
+        using namespace x64;
```

Comment:
It is better to have a single approach either `x64::avx512_core` or `avx512_core`
I would suggest to remove `using namespace x64;` and change everywhere on `x64::`

Reply:
done